src/components/date/date.spec.js

```diff
@@ -473,6 +473,7 @@ describe("Date", () => {
       "12/42/3213",
       "foo",
       "1/01.11",
+      "29/02/2019",
```

Comment:
I think worth adding in a `31/04/2019` for a month with 30 days as well

Reply:
Ah yeah great shout

Reply:
also a great shout 👍